geode-gfsh/src/main/java/org/apache/geode/management/internal/cli/commands/ShowMissingDiskStoreCommand.java

```diff
@@ -95,7 +96,8 @@ private ResultModel toMissingDiskStoresTabularResult(
     ResultModel result = new ResultModel();
 
     boolean hasMissingDiskStores = missingDiskStores.length != 0;
```

Comment:
actually you can get rid of this variable and inline this too, to make it symetric.